webapp/src/components/views/BackendProbe.tsx

```diff
@@ -18,55 +20,61 @@ interface IMaintenance {
     note: string | null | undefined;
 }
 
-export const BackendProbe: FC<IData> = ({ uri, onBackendFound }) => {
+export const BackendProbe: FC<IData> = ({ onBackendFound }) => {
     const classes = useSharedClasses();
     const dispatch = useAppDispatch();
     const { isMaintenance } = useAppSelector((state: RootState) => state.app);
-    const healthUrl = new URL('healthz', uri);
-    const migrationUrl = new URL('maintenanceStatus', uri);
+    const healthUrl = new URL('healthz', BackendServiceUrl);
+    const migrationUrl = new URL('maintenanceStatus', BackendServiceUrl);
 
     const [model, setModel] = useState<IMaintenance | null>(null);
 
     useEffect(() => {
         const timer = setInterval(() => {
-            const fetchHealthAsync = async () => {
-                const result = await fetch(healthUrl);
-
-                if (result.ok) {
+            const onBackendFoundWithAuthCheck = () => {
+                if (!AuthHelper.getAuthConfig()) {
+                    // if we don't have the auth config, re-render the app:
+                    renderApp();
+                } else {
+                    // otherwise, we can load as normal
                     onBackendFound();
                 }
             };
 
-            const fetchMaintenanceAsync = async () => {
-                const result = await fetch(migrationUrl);
+            const fetchHealthAsync = async () => {
+                const result = await fetch(healthUrl);
 
-                if (!result.ok) {
-                    return;
+                if (result.ok) {
+                    onBackendFoundWithAuthCheck();
                 }
+            };
 
-                // Parse json from body
-                result
-                    .json()
+            const fetchMaintenanceAsync = () =>
+                fetch(migrationUrl)
+                    .then((response) => response.json())
                     .then((data) => {
-                        // Body has payload.  This means the app is in maintenance
+                        // Body has payload. This means the app is in maintenance
                         setModel(data as IMaintenance);
+                        return true;
```

Comment:
Should this be returning true here? If the body returns a payload, that's enough to verify that the backend is up so we should not need to probe.